storage/include/object_store/object_store_client.hpp

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 VMware, all rights reserved
+
+#pragma once
+
+#include <memory>
+#include "storage/db_interface.h"
+#include <chrono>
+
+namespace concord {
+namespace storage {
+
+struct S3_StoreConfig {
+  std::string bucketName;  // assuminh pre configured, no need to create
+  std::string url;         // from the customer
+  std::string protocol;    // currently tested with HTTP
```

Comment:
nit - the comment refers to an implementation detail, so maybe it shouldn't be in the header

Reply:
yep. will fix